build.gradle.kts

```diff
@@ -2,12 +2,15 @@ group = "io.provenance.objectstore.gateway"
 version = project.property("version")?.takeIf { it != "unspecified" } ?: "1.0-SNAPSHOT"
 
 plugins {
-	Plugins.Idea.addTo(this)
-	Plugins.NexusPublishing.addTo(this)
-	Plugins.MavenPublish.addTo(this)
-	Plugins.Signing.addTo(this)
-
-	kotlin("jvm") version "1.6.10"
+	kotlin("jvm")
+//	kotlin("plugin.spring") version "1.6.10" apply false
```

Comment:
:yee: with plugin management in settings.gradle no versions needed anywhere except if you want to do some weird jazz in the buildSrc directory.  Then it gets... complicated